spec/openxml/elements/run_properties_spec.rb

```diff
@@ -16,6 +16,18 @@
     end
   end
 
+  context "with font color" do
+    subject { described_class.new(font_color: "929292") }
+    it do
+      is_expected.to generate_tag(<<~XML
+                                  <a:rPr>
+                                    <a:srgbClr val='929292'/>
```

Comment:
I'm not sure how to set the background behind text. I've only figured out the whole slide.